docs/writing-policies/spec/host-capabilities/03-container-registry.md

```diff
@@ -79,3 +79,113 @@ the payload would be the following ones:
+
+### Authentication
+
+Interactions with private registries require the Kubewarden policy host to
+authenticate against the remote registry.
+
+The policy host will use the same set of credentials used to fetch policies
+from the remote registry.
+
+### Communication protocol
+
+This is the description of the waPC protocol used to expose this capability:
+
+<table>
+<tr>
+<th> waPC function name </th> <th> Input payload </th> <th> Output payload </th>
+</tr>
+
+<tr>
+<td>
+
+`v1/image_index`
+
+</td>
+<td>
+
+```hcl
+# OCI URI - JSON encoded string
+string
+```
+
+</td>
+
+<td>
+
+```hcl
+{
+    "schemaVersion": 2,
+        "mediaType": "application/vnd.oci.image.index.v1+json",
+        "manifests": [
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:7d5e84b9314ba7058bfa209881919146ffc4a89c5ba14cfa8270a18c8d418c44",
+            "size": 1048,
+            "platform": {
+                "architecture": "amd64",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:ae1e98105555f3983496c15c70dafc87639b77830953d6470694b47e0e063e25",
+            "size": 1048,
+            "platform": {
+                "architecture": "arm64",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:e9fc0f86e26366abf29fd29e8c09d00df717f9038fd4298eaa39a0a2b4361fa4",
+            "size": 566,
+            "annotations": {
+                "vnd.docker.reference.digest": "sha256:7d5e84b9314ba7058bfa209881919146ffc4a89c5ba14cfa8270a18c8d418c44",
+                "vnd.docker.reference.type": "attestation-manifest"
+            },
+            "platform": {
+                "architecture": "unknown",
+                "os": "unknown"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:e8904ebb2841dc19ae458436eb01ddea3e8d6ea653c7d9476537f4029b1f45a2",
+            "size": 566,
+            "annotations": {
+                "vnd.docker.reference.digest": "sha256:ae1e98105555f3983496c15c70dafc87639b77830953d6470694b47e0e063e25",
+                "vnd.docker.reference.type": "attestation-manifest"
+            },
+            "platform": {
+                "architecture": "unknown",
+                "os": "unknown"
+            }
+        }
+    ]
+}
+```
+
+</td>
+</tr>
+
+</table>
+
+For example, when requesting the manifest digest of the `ghcr.io/kubewarden/policy-server:v1.10.0` image,
+the payload would be the following ones:
+
+* Input payload: `"ghcr.io/kubewarden/policy-server:v1.10.0"`
+* Output payload: The same of the data used in the table above. Which follows
+the [OCI Image index specification](https://github.com/opencontainers/image-spec/blob/main/image-index.md)
```

Comment:
nitpick, the result depends on what the remote image has. It could be a simple manifest, an image index or even a legacy manifest.

I would say something like:

```suggestion
Output payload: the body of the successful response obtained from the registry
```